### ISSUE TITLE
[Accessibility] Known accessibility issues

### ISSUE THREAD

**Issue by HRemonen** [HRemonen/hremonen-site#34]
- [ ] SC 1.4.11 - Non-text Contrast (Level AA): The CTA color `#6ae255` fails contrast requirement of 3:1, this was conscious decision.
- [ ] (PERHAPS) SC 2.4.7 - Focus Visible (Level AA) and SC 1.3.1 Info and Relationships (Level A): The main menu is inside a dialog element which might prevent the menu to be opened using the menu shortcut using assistive technology.
- [ ]  SC 1.4.11 - Non-text Contrast (Level AA): Focus indicators fail to provide sufficient color contrast on certain elements on the page.
